ajax/libs/vue-clickaway/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "vue-clickaway",
+  "description": "Reusable clickaway directive for reusable Vue.js components",
+  "version": "2.1.0",
+  "filename": "vue-clickaway.min.js",
+  "author": {
+    "name": "Denis Karabaza",
+    "email": "denis.karabaza@gmail.com"
+  },
+  "keywords": [
+    "click",
+    "clickaway",
+    "reusable",
+    "vue",
+    "vue-directive",
+    "vuejs"
+  ],
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/simplesmiler/vue-clickaway.git"
+  },
+  "npmName": "vue-clickaway",
```

Comment:
There are 15 releases on GitHub but 13 on NPM. Any reason to use npm auto-update config?